sources/packages/web/src/services/http/common/HttpBaseClient.ts

```diff
@@ -39,6 +39,19 @@ export default abstract class HttpBaseClient {
     }
   }
 
+  protected async getCallTyped<T>(url: string, authHeader?: any): Promise<T> {
+    try {
+      const response = await this.apiClient.get(
+        url,
+        authHeader ?? this.addAuthHeader(),
+      );

```

Comment:
Is it possible to explore the possibility of implementing an interceptor to inject the required auth header(s) instead of passing it as an argument in this generic function?